docs/core/additional-tools/dotnet-svcutil-guide.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Microsoft WCF dotnet-svcutil Tool
+description: An overview of the Microsoft WCF dotnet-svcutil Tool that adds functionality for .NET Core and ASP.NET Core projects, similar to the WCF svcutil tool for .NET Framework projects.
+author: mlacouture
+ms.author: mlacouture
+ms.date: 05/31/2018
+---
+# Microsoft WCF dotnet-svcutil Tool
+
+The Windows Communication Foundation (WCF) **dotnet-svcutil** tool is a .NET Core CLI tool that retrieves metadata from a web service on a network location or from a WSDL file, and generates a WCF class containing client proxy methods that access the web service operations.
+
+Similar to the [**Service Model Metadata - svcutil**](/dotnet/framework/wcf/servicemodel-metadata-utility-tool-svcutil-exe) tool for .NET Framework projects, the **dotnet-svcutil** is a command line tool for generating a web service reference compatible with .NET Core and .NET Standard projects.
+
+The **dotnet-svcutil** tool is an alternative option to the [**WCF Web Service Reference**](/dotnet/core/additional-tools/wcf-web-service-reference-guide) Visual Studio connected service provider which first shipped with Visual Studio 2017 v15.5. The **dotnet-svcutil** tool as a .NET Core CLI tool, is available cross-platform on Linux, MacOS, and Windows.
+
+> [!IMPORTANT]
+> You should only reference services from a trusted source. Adding references from an untrusted source may compromise security. 
+
+## Prerequisites
+
+* [.NET Core SDK](https://www.microsoft.com/net/download) v1.0.4 or later versions
+* Your favorite code editor
+
+## Getting Started
+
+The following example walks you through the steps required to add a web service reference to a .NET Core console project and invoke the service.  You will create a .NET Core console application named _HelloSvcutil_ and will add a reference to a web service that implements the following contract:
```

Comment:
nit: linting rules - leave a new line between paragraphs and code blocks before and after (applies to the rest of the text as well)

remove extra space between sentences

 You will create a .NET Core console application named _HelloSvcutil_ and will add a reference to a web service that implements the following contract -> Create a .NET ... and add a reference...